core/src/main/kotlin/com/katanox/tabour/Tabour.kt

```diff
@@ -25,7 +25,7 @@ import kotlinx.coroutines.newFixedThreadPoolContext
  * By default, the container will allocate 4 threads which can be configured using [Configuration]
  */
 class Tabour internal constructor(val config: Configuration) {
-    private val registries: MutableSet<Registry<*>> = mutableSetOf()
+    val registries: MutableSet<Registry<*>> = mutableSetOf()
```

Comment:
Is this accessible from outside the `Tabour` file?

Reply:
ye i will bring it back